poc/README.md

```diff
@@ -19,10 +19,26 @@ Version 3.20.0 or later is required.
 
 ## Generating test vectors
 
```

Comment:
Remove `generate_test_vectors` from the Makefile (see above) and update these instructions.

Reply:
Done.